definitions/2.2.0/APIKeyHTTPSecurityScheme.json

```diff
@@ -29,10 +29,10 @@
   },
   "patternProperties": {
     "^x-[\\w\\d.\\-_]+$": {
-      "$ref": "specificationExtension.json"
+      "$ref": "http://asyncapi.com/draft-07/schemas/specificationExtension"
     }
   },
   "additionalProperties": false,
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "APIKeyHTTPSecurityScheme.json"
+  "$id": "http://asyncapi.com/draft-07/schemas/APIKeyHTTPSecurityScheme"
```

Comment:
It should be reversed to the `.../schemas/draft-07/...` because `draft-07` points to schema version, not to AsyncAPI version 😄

Reply:
On the other hand, it can stay because $id only work in JSON Schemes... Mondays 😅

Reply:
Just an example URI to show the change 😄 Probably `http://asyncapi.com/schemas/APIKeyHTTPSecurityScheme` or `http://asyncapi.com/schemas/2.2.0/APIKeyHTTPSecurityScheme` would be more suitable.

Reply:
- The `http://asyncapi.com/schemas/APIKeyHTTPSecurityScheme` will point to a schema that will not be changed, so it is better to have some version.
- The `http://asyncapi.com/schemas/2.2.0/APIKeyHTTPSecurityScheme` points to the AsyncAPI version, to we will need to update (copy?) schemas on each release 😄

Reply:
We just need to resolve https://github.com/asyncapi/spec-json-schemas/pull/128#issuecomment-981344308